package.json

```diff
@@ -21,7 +21,7 @@
     "build:digital-payments": "yarn --cwd integration-libs/digital-payments run build:schematics && ng build digital-payments --configuration production",
     "build:epd-visualization": "yarn --cwd integration-libs/epd-visualization run build:schematics && ng build epd-visualization --configuration production",
     "build:order": "yarn --cwd feature-libs/order run build:schematics && ng build order --configuration production",
-    "build:libs": "ng build core --configuration production && ng build storefrontlib --configuration production && concurrently --kill-others-on-fail yarn:build:schematics yarn:build:user && yarn build:cart && concurrently --kill-others-on-fail yarn:build:order yarn:build:checkout yarn:build:asm yarn:build:tracking && concurrently --kill-others-on-fail yarn:build:organization yarn:build:product yarn:build:product-configurator yarn:build:storefinder yarn:build:cdc && concurrently --kill-others-on-fail yarn:build:smartedit yarn:build:qualtrics yarn:build:assets yarn:build:cds yarn:build:digital-payments yarn:build:epd-visualization && yarn build:setup",
+    "build:libs": "ng build core --configuration production && ng build storefrontlib --configuration production && concurrently --kill-others-on-fail yarn:build:schematics yarn:build:user && yarn build:cart && yarn build:order && concurrently --kill-others-on-fail yarn:build:checkout yarn:build:asm yarn:build:tracking && concurrently --kill-others-on-fail yarn:build:organization yarn:build:product yarn:build:product-configurator yarn:build:storefinder yarn:build:cdc && concurrently --kill-others-on-fail yarn:build:smartedit yarn:build:qualtrics yarn:build:assets yarn:build:cds yarn:build:digital-payments yarn:build:epd-visualization && yarn build:setup",
```

Comment:
Is this change really related to current PR?

Reply:
no, but I was experiencing the build failing due to the fact that `order` lib must be built before checkout.

Reply:
Strange other folks don't experience it. Can you reproduce the same problem also in fresh `develop`?
If it's a _real issue_, for github history clarity, it would be nice to fix it in another PR.

Reply:
I don't remember what I was doing at that moment to experience it. 
If you insist, we can revert this line?

Reply:
If possible, let's revert it please